src/main/webapp/app/exercises/text/participate/text-editor.component.ts

```diff
@@ -168,17 +169,10 @@ export class TextEditorComponent implements OnInit, OnDestroy, ComponentCanDeact
     }
 
     /**
-     * Find "Unreferenced Feedback" item for Result, if it exists.
+     * Check whether or not a result exists and if, returns the unreferenced feedback of it
      */
-    get unreferencedFeedback(): Feedback[] | null {
-        if (this.result && this.result.feedbacks && Array.isArray(this.result.feedbacks)) {
-            const feedbackWithoutReference = this.result.feedbacks.filter(
-                (feedbackElement) => feedbackElement.reference == undefined && feedbackElement.type === FeedbackType.MANUAL_UNREFERENCED,
```

Comment:
This removes the last usage of `FeedbackType`, so we get an unused import warning in this file too.